README.md

```diff
@@ -37,7 +37,7 @@ objectql({ a: 'a', b: 'b' }, query); // returns { a: 'a' }
 objectql([{ a: 'a', b: 'b' }, { b: 'b', c: 'c' }], query); // returns [{ a: 'a' }, {}]
 ```
 
-__query__ must be an object, and only the keys where the value is `true` will be used. If an object is used as a key's value it will apply that object as the `query` param to that part of the source object recursively.
+__query__ must be a concise string of key names, like a 'simple' graphql query.
```

Comment:
I like that you've updated all the docs to use the new style - but we should keep some documentation describing the object way.

What about:
```
__query__ Can be either a string or an object, see below for examples.
```

Then in the `// valid query key values` docs, we have examples of each query in both string + object form - the rest of the examples can be updated to the new string form like you've done.